guides/schema/class_based_api.md

```diff
@@ -309,12 +309,16 @@ class Types::TodoList < Types::BaseObject
   field :name, String, "The unique name of this list", null: false
   field :is_completed, String, "Completed status depending on all tasks being done.", null: false
   # Related Object:
-  field :owner, Types::User, "The creator of this list", null: false
+  field :owner, Types::User, "The creator of this list", null: false do
   # List field:
-  field :viewers, [Types::User], "Users who can see this list", null: false
+  field :viewers, [Types::User], "Users who can see this list", null: false do |f|
+    # Field definitions still accept a block but now provide the field class
```

Comment:
Added this, not sure if it deserves it's own section explaining field definition?